src/fusor/harvester.py

```diff
@@ -293,7 +293,9 @@ def __init__(
 
         civic.load_cache(local_cache_path=local_cache_path, on_stale="ignore")
         self.translator = CIVICTranslator(fusor=fusor)
-        self.fusions_list = None
+
+        # Load in accepted fusion variants from CIViC
+        self.fusions_list = civic.get_all_fusion_variants(include_status="accepted")
```

Comment:
Should we allow users the option to select the status and set the default to accepted?